flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -69,7 +70,13 @@
 
 	private static final Logger LOG = LoggerFactory.getLogger(NetworkEnvironment.class);
 
-	private static final String METRIC_GROUP_NETWORK = "Network";
+	@SuppressWarnings("DeprecatedIsStillUsed")
+	@Deprecated private static final String METRIC_GROUP_NETWORK_DEPRECATED = "Network";
+	private static final String METRIC_GROUP_SHUFFLE = "Shuffle";
+	private static final String METRIC_GROUP_NETTY = "Netty";
+	private static final String METRIC_GROUP_OUTPUT = "Output";
+	private static final String METRIC_GROUP_INPUT = "Input";
+	private static final String METRIC_GROUP_BUFFERS = "Buffers";
```

Comment:
There are already too many metrics here and it seems hard to distinguish them if not focusing on the below implementations. Could we give some comments or other forms to clearly know the architecture of these metrics? 
At least to distinguish the task-level metrics and global metrics.

Reply:
It is more clearly now 👍